media/js/firefox/sticky-promo.js

```diff
@@ -43,20 +53,40 @@
         };
 
         StickyPromo.show = function (){
-        // Open promo
-            Mzp.StickyPromo.open();
+            var stickyPromoShowInit = false;
+            document.addEventListener('scroll', function () {
+                if (!stickyPromoShowInit) {
```

Comment:
Seeing as this code only runs once, might it make sense to remove the scroll event handler once the promo is shown?